modules/nf-core/samtools/idxstats/main.nf

```diff
@@ -1,6 +1,6 @@
 process SAMTOOLS_IDXSTATS {
     tag "$meta.id"
-    label 'process_low'
+    label 'process_single'
```

Comment:
This seems to be in conflict with the option given to the command `--threads ${task.cpus-1}`. The -1 doesn't make sense to me in general but are you saying that `samtools idxstats` cannot operate multithreaded or it doesn't need to? In that case I would also remove the `--threads` option or hard code it to 1.

Reply:
Okay, going with your take, I guess this is fine.

Reply:
I can confirm the `--threads` option is ignored in this tool. The `-1` value was to account for the 2 cpu's allotted through `process_low`. I actually should've removed it, good catch @Midnighter